src/app/progress/Objective.tsx

```diff
@@ -62,10 +64,17 @@ export default function Objective({
   }
 
   const isBoolean = isBooleanObjective(objectiveDef, completionValue);
+  const showAsCounter =
+    isTrialsPassage &&
+    objectiveDef.allowOvercompletion &&
+    objectiveDef.progressDescription === 'Rounds Won';
```

Comment:
this will also not work with other locales

Reply:
isWinsObjective

Reply:
if you need string matching consider a PR to d2ai for everything required here... and making a json file to check against instead.

https://github.com/DestinyItemManager/d2-additional-info

Reply:
Thanks for that. For this one I could get it without using a string comparison as it is different to the other objectives within the item, so now I don't have anymore string comparisons.